### PR TITLE
[FEAT] Goal Command Application

### PR DESCRIPTION
**Pull request by sueelly**
## #️⃣ Related Issue

Linked Feature backlog: [issue#3](https://github.com/dnd-side-project/dnd-12th-2-backend/issues/3)

## 📝 Purpose of PR

Implement Goal Command (Change of data)'s logic and verify it with unit test

### Applying Tdd
1. dto/model
2. use case
3. service with no body
4. repository (also interface)
5. fixture
6. unit test
7. implement service

## Checklist

-   [x] Does commit messages follow the convention?
-   [x] Are variable names brief but descriptive?
-   [x] Is the code clean and easy to understand?
-   [x] If a new feature has been added, or a bug fixed, has a test been added to confirm good behavior?
-   [x] Does the test successfully test edge and corner cases?

## 💬 Review Requirements (Optional)

Is there anything you can praise about this PR? Start with praise.
Make a review, leaving kind comments and suggesting changes where needed (to resolve the above).

### CODE REVIEW

File: src/test/java/ac/dnd/dodal/application/goal/service/GoalCommandServiceTest.java
Comment (5hseok) [inline src/test/java/ac/dnd/dodal/application/goal/service/GoalCommandServiceTest.java:1]: test 코드들이 깔끔하게 잘 정리되어 있는 것 같아요! 참고해서 저도 작성하겠습니다!

**Review by 5hseok — APPROVED** [review]

File: src/main/java/ac/dnd/dodal/application/goal/service/GoalCommandService.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+import ac.dnd.dodal.application.goal.usecase.CreateGoalUseCase;
+import ac.dnd.dodal.application.goal.usecase.AchieveGoalUseCase;
+import ac.dnd.dodal.application.goal.usecase.DeleteGoalUseCase;
+import ac.dnd.dodal.application.goal.repository.GoalCommandRepository;
+
+@Service
+@Transactional
+@RequiredArgsConstructor
+public class GoalCommandService
+        implements CreateGoalUseCase, AchieveGoalUseCase, DeleteGoalUseCase {
```
Comment (5hseok) [inline src/main/java/ac/dnd/dodal/application/goal/service/GoalCommandService.java:23]: 하나의 서비스에서  여러 개의 UseCase를 구현하게 되면 단일책임원칙에서 벗어나지 않을까요? 그리고 CQRS에서는 UseCase 사용 시 Command 패턴을 사용해서 UseCase의 메서드명을 통일한다고 알고 있는데, 이 부분에 대해서는 어떻게 생각하시나요?

**Review by 5hseok — APPROVED** [review]

File: src/main/java/ac/dnd/dodal/application/goal/service/GoalCommandService.java (hunk 1 above)
Comment (sueelly) [reply src/main/java/ac/dnd/dodal/application/goal/service/GoalCommandService.java:23]: cqrs로 넘어갔을 때 합칠 필요성이 생기면 합치는 것으로 결론
